ngraph/core/src/op/util/gather_base.cpp

```diff
@@ -52,26 +52,26 @@ void op::util::GatherBase::validate_and_infer_types()
             axis_pshape);
     }
 
-    int64_t batch_dims = get_batch_dims(); // will not be converted to positive if axis is not set
+    int64_t batch_dims = get_batch_dims(); // will not be normalized if indices rank is dynamic
     if (is_axis_set())
     {
         int64_t axis = get_axis();
+
+        // batch_dims, axis both can e positive by default or if data_rank & indices_rank are
```

Comment:
Please, fix typo - `...both can be positive by...` maybe?.

Reply:
will fix this together with other comments which I'm sure other will also leave

Reply:
fixed